pkg/app/app.go

```diff
@@ -221,6 +222,30 @@ func Convert(opt kobject.ConvertOptions) {
 
 	komposeObject.Namespace = opt.Namespace
 
+	// convert env_file from absolute to relative path
+	for _, service := range komposeObject.ServiceConfigs {
+		if len(service.EnvFile) <= 0 {
+			continue
+		}
+		for i, envFile := range service.EnvFile {
+			if !filepath.IsAbs(envFile) {
+				continue
+			}
+
+			workDirAbs, err := filepath.Abs(filepath.Dir(opt.InputFiles[0]))
+			if err != nil {
+				log.Fatalf(err.Error())
+			}
+
+			relPath, err := filepath.Rel(workDirAbs, envFile)
+			if err != nil {
+				log.Fatalf(err.Error())
+			}
+
+			service.EnvFile[i] = strings.Replace(relPath, "\\", "/", -1)
```

Comment:
Can you please clarify more why we need this, because what you added in  the`kubernetes.go` file should solve the problem.

Reply:
Sure,

**1)** To address the issue of excessively long names, I transitioned from absolute paths to relative ones.

**2)** A substitution occurs within the environment name. Hence, even when using relative paths, this substitution remains necessary. 
- Refer to the code [k8sutils.go#L855](https://github.com/kubernetes/kompose/blob/b83d4d4094ff887c087aa48f8dbab9803afe764c/pkg/transformer/kubernetes/k8sutils.go#L855). 
- It replaces "/" with "-". 
- Since Windows paths utilize "\\", this won't resolve the environment name issue.